core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -309,9 +309,13 @@ oppia.directive('conversationSkin', [
           };
 
           $scope.isCorrectnessFooterEnabled = function() {
-            return (
+            var correctnessFeedbackEnabled = (
               $scope.answerIsCorrect && $scope.isCorrectnessFeedbackEnabled() &&
               PlayerPositionService.hasLearnerJustSubmittedAnAnswer());
+            if (correctnessFeedbackEnabled) {
```

Comment:
This is a getter method. It shouldn't "do" things. So currently it has a side-effect.

Can we move the scroll to the caller or the part of the code which is responsible for actively doing things?

Reply:
Done, removed this, and added fade out.